delft/packet-importer.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
+let
+  importer = pkgs.callPackage ../hydra-packet-importer { };
+in
+{
+  deployment.keys."hydra-packet-import.json" = {
+    keyFile = ../hydra-packet-import.json;
+    user = "hydra-packet";
+  };
+
+  users.extraUsers.hydra-packet =
+    { description = "Hydra Packet Machine Importer";
+      group = "hydra";
+    };
+
+  system.activationScripts.hydra-packet = lib.stringAfter [ "users" ]
+    ''
+      mkdir -m 0755 -p /var/lib/hydra/packet-import
+      chown hydra-packet.hydra /var/lib/hydra/packet-import
+      if [ ! -f /var/lib/hydra/packet-import ]; then
+        touch  /var/lib/hydra/packet-import/machines
+        chown hydra-packet.hydra /var/lib/hydra/packet-import/machines
+        chmod 0644 /var/lib/hydra/packet-import/machines
+      fi
+      chown hydra-packet.hydra /var/lib/hydra/packet-import
```

Comment:
this is the same as in L19 too.

Reply:
Replaced with the tmpfiles rule, right? Pretty cool.